src/include/ceph_features.h

```diff
@@ -184,7 +184,8 @@ DEFINE_CEPH_FEATURE(60, 1, OSD_RECOVERY_DELETES) // *do not share this bit*
 DEFINE_CEPH_FEATURE(61, 1, CEPHX_V2)         // 4.19, *do not share this bit*
 
 DEFINE_CEPH_FEATURE(62, 1, RESERVED)           // do not use; used as a sentinel
-DEFINE_CEPH_FEATURE_DEPRECATED(63, 1, RESERVED_BROKEN, LUMINOUS) // client-facing
+DEFINE_CEPH_FEATURE_RETIRED(63, 1, RESERVED_BROKEN, LUMINOUS, QUINCY) // client-facing
+// available
```

Comment:
> X-3 == R-3 == Octopus

I don't think `RESERVED_BROKEN` was tied to Octopus or the "X-3" rule.   But deciding that it has been long enough for `OSD_SNAPMAPPER` workaround to go away entirely is fine too :-)

Reply:
If we can retire this bit, let's go for it.

Just to the record the meaning of the macros' params:

```cpp
// this bit is ignored but still advertised by release *when*
#define DEFINE_CEPH_FEATURE_DEPRECATED(bit, incarnation, name, when) \
        const static uint64_t DEPRECATED_CEPH_FEATURE_##name = (1ULL<<bit); \
        const static uint64_t DEPRECATED_CEPH_FEATUREMASK_##name =              \
                (1ULL<<bit | CEPH_FEATURE_INCARNATION_##incarnation);

// this bit is ignored by release *unused* and not advertised by
// release *unadvertised*
#define DEFINE_CEPH_FEATURE_RETIRED(bit, inc, name, unused, unadvertised)
```